fruit-app/README.md

```diff
@@ -2,9 +2,22 @@
 
 Very basic CRUD application for fruit
 
-## Running locally
+## Local Development Setup
 
 - Install dependencies, `npm install`
 - Run the server, `npm run`
 
 When `client.js` has been modified, run `npm run client:build`
+
+Application is then accessible on http://localhost:8080/ if `process.env.PORT` is undefined.
+
+## Building and Running Application in Docker locally
+
+```
+# sudo is only necessary in CentOS, Fedora, or RHEL systems
+sudo docker build -t quay.io/integreatly/fruit-crud-app .
```

Comment:
I would omit the `sudo` from these commands

Reply:
Cool, sudo have been ommitted from these commands